src/app/globals.css

```
@@ -2,20 +2,66 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+  font-family: 'Saiyan-Sans';
+  src: url('/fonts/Saiyan-Sans.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+body {
+  @apply bg-black min-h-screen;
+  overflow-x: hidden;
+}
+
 :root {
-  --background: #ffffff;
-  --foreground: #171717;
+  --grid-color: rgba(16, 185, 129, 0.1);
+  --grid-size: 50px;
+  --background-size: 15px;
+  --particle-animation-duration: 20s;
 }
 
-@media (prefers-color-scheme: dark) {
-  :root {
-    --background: #0a0a0a;
-    --foreground: #ededed;
-  }
+/* グリッドライン背景 */
+.grid-lines {
```

ぐわああ
[nits]
我ながらやらかした感満載なのですが、className にキャメルケースとケバブケースが混在していますね、、思いっきり私もやらかしていた。。😂

とりあえず、tailwilnd CSS と見分けるためにCSS Module 含めた className はキャメルケースにする感じでどうですか？

https://github.com/Kazuki0320/sns-scouter/blob/main/src/components/ui/rotatingArcs.module.css

https://github.com/Kazuki0320/sns-scouter/blob/main/src/components/ui/experimentSpeechBubble.module.css

```suggestion
.gridLines {
```

---

😳😳😳😳

> tailwilnd CSS と見分けるためにCSS Module 含めた tailwilnd CSS 以外の className はキャメルケースにする感じでどうですか？

賛成！